frontend/src/stimulus/controllers/op-application.controller.ts

```diff
@@ -8,11 +8,11 @@ export class OpApplicationController extends ApplicationController {
 
   dynamicTargetConnected(target:HTMLElement) {
     const controllers = (target.dataset.controller as string).split(' ');
+    const registered = this.application.controllers.map((controller) => controller.identifier);
 
     controllers.forEach((controller) => {
       const path = this.derivePath(controller);
-
-      if (!this.loaded.has(controller)) {
+      if (!registered.includes(controller) && !this.loaded.has(controller)) {
```

Comment:
This had to be added as the same form now has a static and dynamic controller, so we have to skip those that are already registered (I'm wondering this hasn't occurred before)